web/frontend/src/components/movement-sensor/index.svelte

```diff
@@ -106,6 +116,35 @@ useConnect(() => {
               {altitudeM?.toFixed(2)}
             </td>
           </tr>
+          {#if accuracy?.positionNmeaGgaFix}
+          <tr>
+            <th class="border border-medium p-2"> NMEA Fix Quality </th>
+            <td class="border border-medium p-2">
+              {accuracy.positionNmeaGgaFix}
+              {#if accuracy.positionNmeaGgaFix === 1 || accuracy.positionNmeaGgaFix === 2}
+                <div>expect 1m-5m accuracy</div>
+              {/if}
+              {#if accuracy.positionNmeaGgaFix === 4 || accuracy.positionNmeaGgaFix === 5}
+                <div>expect 2cm-50cm accuracy</div>
+              {/if}
```

Comment:
This is a lot of text to fit in the cell. Could we use a [Tooltip](https://prime.viam.com/?path=/docs/elements-tooltip--docs) on an info [Icon](https://prime.viam.com/?path=/docs/elements-icon--docs) to convey this?